mani_skill/utils/wrappers/record.py

```diff
@@ -743,6 +743,15 @@ def flush_video(
                 video_name = "{}".format(self._video_id)
                 if suffix:
                     video_name += "_" + suffix
+                if self._avoid_overwriting_video:
```

Comment:
what is this for? Either document or remove it.

Reply:
documented